harmonizer/types.ts

```diff
@@ -46,6 +48,7 @@ export type HarmonyRelease = {
 	language?: Language;
 	script?: ScriptFrequency;
 	status?: ReleaseStatus;
+	types?: Set<ReleaseGroupType>;
```

Comment:
Another reason to avoid a set here is that we might want to have the merge algorithm sort the values with primary type first. I'll look into this when redoing the merging.

Reply:
Fixed as part of 0f08a807935dde96f4ec66dcc5917aff33703a4e